### PR TITLE
Feat/automl interface

### PR DESCRIPTION
**Pull request by oliveirabruno92**
Added a client for automl API with three method: 
- List jobs
- Retrieve job
- Run job

### CODE REVIEW

File: examples/automl_client.ipynb [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+    "from elemeno_ai_sdk.ml.automl.client import AutoMLClient"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "API_KEY = \"eyJhbGciOiJSUzI1NiIsImtpZCI6ImVsZW1lbm8tc2Fhcy1hcHBzLWtleS1pZCIsInR5cCI6IkpXVCJ9.eyJhY2NvdW50Ijoib2xpdmVpcmEudWZyajExQGdtYWlsLmNvbSIsInN1YiI6ImRldmVsb3BtZW50QGVsZW1lbm8uYWkiLCJpYXQiOjE2OTExNzMzODAsImV4cCI6MTY5NjA5ODE3MywiaXNzIjoiZGV2ZWxvcG1lbnRAZWxlbWVuby5haSJ9.eh4FjO_ym_G6PHzmbb2sW7uOVgNWtf_IGcuCTC10m4RbdC5fXhsOdcBO3bzfG8zIBmxOhJ1rGWPXbqOfx0_XELIeasvak7BhhuknpSgOPhmSjtH-jAhVbzVIn2En83vCk_kKzTqk9J2KwbbwxdTxERw4iSGuJ1GVsdIefnmtT8SQpEDFta1gblHzjXHWAKU73lmzvX3lUtb8z-6pUyX2KYQEJWsoU1Aep8tDg1UcNj0on1znLzqtM7LPTqy_0vFjWn40lIOck8mJVnXdfZsfQdVt_fnQ8RBPfRCiyrcqh0miKb4-RcsXRdH4VQP69sdMVdcX0p63p54Zx1mDSBQjfg\""
```
Comment (wagnersemantix) [inline examples/automl_client.ipynb:26]: Remove API key from the public repo

File: src/elemeno_ai_sdk/ml/automl/client.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+class AutoMLClient:
+    def __init__(self, env: str, api_key: str) -> None:
+        self.env = env
+        self.api_key = api_key
+
+    @property
+    def base_url(self):
+        base_url = None
+        if self.env == "prod":
+            base_url = "https://c3po.ml.semantixhub.com"
```
Comment (wagnersemantix) [inline src/elemeno_ai_sdk/ml/automl/client.py:16]: One suggestion is to move these definitions outside the class scope to the module level, like:
`PROD_ENV = "https://c3po.ml.semantixhub.com"`
and then reference it inside the class

File: src/elemeno_ai_sdk/ml/automl/client.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        else:
+            raise ValueError("Invalid environment. Please use dev or prod.")
+        return base_url
+
+    @property
+    def headers(self):
+        return {"Content-Type": "application/json", "x-api-key": self.api_key}
+
+    async def list_jobs(self):
+        async with aiohttp.ClientSession(headers=self.headers) as session:
```
Comment (wagnersemantix) [inline src/elemeno_ai_sdk/ml/automl/client.py:28]: These session blocks in `list_jobs`, `run_jobs` and `get_jobs` look the same, perhaps move them to a single function to improve readability?

**Review by wagnersemantix — CHANGES_REQUESTED** [review]

**Review by wagnersemantix — APPROVED** [review]

File: src/elemeno_ai_sdk/ml/automl/client.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+from typing import Dict
+
+import aiohttp
+
+PROD_URL = "https://c3po.ml.semantixhub.com"
+DEV_URL = "https://c3po-stg.ml.semantixhub.com"
+
+
+class AutoMLClient:
+    def __init__(self, env: str, api_key: str) -> None:
```
Comment (lucasbmiguel) [inline src/elemeno_ai_sdk/ml/automl/client.py:12]: I suggest you leave the default value for env = PROD

**Pull request merged**